prometheus/manifests/prometheus/010-prometheus-cr.yaml

```diff
@@ -9,16 +9,22 @@ spec:
     requests:
       memory: 400Mi
   serviceAccountName: prometheus
-  serviceMonitorNamespaceSelector: {}
-  serviceMonitorSelector: {}
-#  securityContext:
-#    fsGroup: 2000
-#    runAsNonRoot: true
-#    runAsUser: 1000
-#  storage:
-#    volumeClaimTemplate:
-#      spec:
-#        storageClassName: fast
-#        resources:
-#          requests:
-#            storage: 10Gi
+  serviceMonitorSelector:
+    matchLabels:
+      app: storageos
+  securityContext:
+    fsGroup: 2000
+    runAsNonRoot: true
+    runAsUser: 1000
+  storage:
+    volumeClaimTemplate:
+      metadata:
+        name: data
+        labels:
+          env: prod
+      spec:
+        accessModes: ["ReadWriteOnce"]
+        storageClassName: fast
+        resources:
+          requests:
+            storage: 1Gi
```

Comment:
a 1Gi disk will soon be filled by Prometheus. I would put something on the lines of 100Gi.